fs/glob.py

```diff
@@ -71,6 +150,8 @@ def match(pattern, path):
     except KeyError:
         levels, recursive, re_pattern = _translate_glob(pattern, case_sensitive=True)
         _PATTERN_CACHE[(pattern, True)] = (levels, recursive, re_pattern)
+    if path and path[0] != "/":
+        path = "/" + path
```

Comment:
Is this the same as https://docs.pyfilesystem.org/en/latest/reference/path.html?#fs.path.abspath ?

Reply:
Only with the current implementation of `abspath`, not with the general idea.